selenium-tests/test_registration.py

```diff
@@ -1,25 +1,43 @@
 import pytest
 from selenium.webdriver.support.select import Select
 import uuid
+import time
 
 # TODO: eventually, make this configurable
 EVENT_NAME = "magstock"
 
+STRIPE_FAKE_CREDIT_CARD = "4242"  # this will be repeated 4 times
+STRIPE_FAKE_EXP_DATE = "12/25"  # MM/YY
```

Comment:
Some day in the distant future we're all going to wonder why these tests started breaking.

It's a bit silly, but actually could you possibly use "current year +1" here instead of a hard-coded date?